Zend/tests/enum/__clone.phpt

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Enum __clone
+--FILE--
+<?php
+
+enum Foo {
+    case Bar;
+
+    public function __clone() {
+    }
+}
+
+?>
+--EXPECTF--
+Fatal error: Enum may not include magic method __clone in %s on line %d
```

Comment:
cannot we add the enum name as well? We tried to be as specific as possible (and as much it makes sense), and I think it's a very useful piece of information.

Reply:
Nit: instead of "may not", PHP uses "cannot" in the overwhelming majority of error messages

Reply:
> Nit: instead of "may not", PHP uses "cannot" in the overwhelming majority of error messages

That's a good point. I'm happy to change that if you think it's worth it?

Reply:
Yeah, it think it's worth it if Ilija doesn't mind it :)

Reply:
For sure, go for it :slightly_smiling_face:

Reply:
For clarification @kocsismate @iluuu1994 the following error message:

```
Enum may not include magic method __clone
```

Becomes the following:

```
Enum YourEnumName cannot include magic method __clone
```

Changing "may not" to "cannot" makes sense, but I understand the lack of class name to be a statement about enums in general, as in, no enum may implement X.